### PR TITLE
Add header changes and add footer to resource list

### PR DESCRIPTION
**Pull request by vzajkov**
## Description
Made header sticky - now stays in view when user scrolls on desktop. Added footer to the resource list for desktop.

## Test Plan
Header stays in view when scrolling.

### CODE REVIEW

File: client/js/mini-app/logo-and-navigation.js [hunk 1]
```diff
@@ -45,10 +45,14 @@ const styles = {
...
 		height: '100px',
-		margin: '0px 134px 0px 42px',
+		padding: '0px 134px 0px 42px',
+		'background-color': '#FFFFFF',
+		'min-width': '90vw',
 		display: 'flex',
 		'align-items': 'center',
 		'justify-content': 'space-between',
+		position: 'fixed',
+		top: '0vh',
```
Comment (karenmrose) [inline client/js/mini-app/logo-and-navigation.js]: These changes will only affect the desktop view. Changes will also need to be made for the mobile view 😄

**Comment by karenmrose** [comment]
@vzajkov The sticky header on the desktop view looks great! It doesn't look like it's sticky on the mobile view though. I added a comment on the file regarding mobile view.

As far as the footer - it's great that it's showing up on all pages now!! We don't need to make the footer "sticky" though. It should show up at the bottom as you scroll to the bottom, but not before you scroll to the bottom. Let me know if you have any questions about that.

**Comment by karenmrose** [comment]
One more thing: the top blue line should also stick with the header as you scroll (i.e. both the header and the blue line should be sticky)

**Comment by vzajkov** [comment]
@karenmrose Fixed styling according to comments, let me know if there are any issues

**Comment by karenmrose** [comment]
Hey @vzajkov would you mind merging in the latest master?

File: client/js/mini-app/header.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ const styles = {
 		display: 'flex',
 		'background-color': '#016454',
 		'border-bottom': '4px solid #3D65F9',
+		'margin-top': '100px',
```
Comment (karenmrose) [inline client/js/mini-app/header.js]: This looks good! We need to make the same change for the mobile view too though.

Comment (vzajkov) [reply client/js/mini-app/header.js]: Good call, thanks -  merged in master and added a 30px margin-top for mobile.

**Comment by karenmrose** [comment]
@vzajkov It looks like there is still a space on the mobile view

Screen Shot 2019-04-13 at 10 42 52 AM

File: client/js/mini-app/header.js (hunk 2 above)
Comment (karenmrose) [reply client/js/mini-app/header.js]: Ok great. If you inspect the header element though, the height is `55px` so the margin should also be `55px` to match

Screen Shot 2019-04-13 at 10 46 43 AM

**Pull request merged**